### PR TITLE
first commit, 10/10 game promiz

### PR DESCRIPTION
**Pull request by ludvigch**
## Changes:

- Added my super cool minigame called "Who should I follow on github?"

## Please tick the checkboxes below to ensure you have done them:

- [x] Read [Contributing.md](https://github.com/linxea/css-animation/blob/master/.github/CONTRIBUTING.md)
- [x] Add an index.html with your css magic in YOUR_FOLDER_NAME. E.g. `/docs/YOUR_FOLDER_NAME/index.html`
- [x] Update README.md a link to your YOUR_FOLDER_NAME. E.g. `## [n. YOUR CSS ANIMATION NAME](https://linxea.github.io/css-animation/YOUR_FOLDER_NAME)`

### CODE REVIEW

File: docs/ludvigch/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <div class="directions">
+        This wheel decides what account you should follow on github!
+  </div>
```
Comment (linxea) [inline docs/ludvigch/index.html]: Let's indent the html tag properly to satisfy my OCD. 😁

**Comment by linxea** [comment]
OMG, you are the first one to follow all the points accurately. ✋high five 🎉I'm so touched. Your example is freaking amazing! Just need to resolve conflict as another merged PR has taken number 2. in README.md.

**Review by linxea — APPROVED** [review]
LGTM

**Comment by ludvigch** [comment]
Yay I did a do! Merged and fixed indentations!

**Comment by linxea** [comment]
YAY MERGED!

**Pull request merged**